cli/create_and_populate_database.bash

```diff
@@ -12,23 +12,23 @@ rm -Rf subscriptions/migrations/*
 rm -Rf tags/migrations/*
 
 
-python manage.py reset_db --noinput
+python3 manage.py reset_db --noinput
```

Comment:
The `python` command will be the right one when running the command in the virtual environment (`venv`). Please roll back